lib/plugins/bayesian-optimization/bloc/bayesian_optimization_hub_bloc.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+// Define events for the bloc
+abstract class BayesianOptimizationHubEvent {}
+
+class BayesianOptimizationHubReloadEvent extends BayesianOptimizationHubEvent {}
+
+// Define states for the bloc
+abstract class BayesianOptimizationHubState {}
+
+class BayesianOptimizationHubInitial extends BayesianOptimizationHubState {}
+
+class BayesianOptimizationHubLoaded extends BayesianOptimizationHubState {}
+
+// Define the bloc
+class BayesianOptimizationHubBloc extends Bloc<BayesianOptimizationHubEvent, BayesianOptimizationHubState> {
+  BayesianOptimizationHubBloc() : super(BayesianOptimizationHubInitial());
+
+  @override
+  Stream<BayesianOptimizationHubState> mapEventToState(BayesianOptimizationHubEvent event) async* {
+    if (event is BayesianOptimizationHubReloadEvent) {
+      yield BayesianOptimizationHubLoaded();
+    }
+  }
```

Comment:
I think this is deprecated and should be replaced by on<Event>: https://github.com/felangel/bloc/issues/2526